website/docs/components/tooltip/partials/code/how-to-use.md

```diff
@@ -0,0 +1,98 @@
+## How to use this component
+
+The HDS tooltip is provided as both a `TooltipButton` component and as an `hds-tooltip` Ember modifier. To be accessible, the element which triggers a tooltip needs to be an interactive element which can receive focus. This means that:
+
+* if the tooltip needs to be applied to a non-interactive element, the `TooltipButton` component should be used as it wraps the opener element with a `button` element.
+* if the tooltip needs to be applied to an element which is already interactive, the `hds-tooltip` mmodifier should be used.
+
+Currently, the tooltip uses [Tippy.js](https://atomiks.github.io/tippyjs/) under the hood for its main functionality.
+
+<!-- use the same heading order from Guidelines -->
+### Content
+
+#### Icon
+```handlebars
+  <Hds::TooltipButton @text="Here is more information">
+    <FlightIcon @name="alert-circle" aria-label="More information" />
+  </Hds::TooltipButton>
+```
+
+#### Inline with text
+```handlebars
+  <p>
+    <strong>Header text</strong>
+    <Hds::TooltipButton @text="Here is more information">
+      <FlightIcon @name="alert-circle" aria-label="More information" />
+    </Hds::TooltipButton>
+  </p>
+```
+
+#### Placement
+
+The tooltip appears at the “top” centered above the opener button content by default. If the tooltip is near any of the edges of the screen, its position will automatically adjust to prevent the tooltip content from being cut off so it remains readable.
+
+Use the `@placement` argument if you would like to use a different starting position for the tooltip vs. the default.
+
+
+```handlebars
+  <Hds::TooltipButton @text="Here is more information" @placement="bottom">
+    <FlightIcon @name="alert-circle" aria-label="More information" />
+  </Hds::TooltipButton>
+```
+
+#### Offset
+
+You can change the offset of the tooltip in relation to the opener element content if needed.
+
+
+```handlebars
+  <Hds::TooltipButton @text="Here is more information" @offset={{array 50 30}}>
+    <FlightIcon @name="alert-circle" aria-label="More information" />
+  </Hds::TooltipButton>
+```
+
+#### Extra Tippy Options
+
+Using `@extraTippyOptions` to enable rich tooltip text.
```

Comment:
```suggestion
You can use `@extraTippyOptions` to enable rich tooltip text.
```

Reply:
Also, probably this sub-section should be expanded a bit: `@extraTippyOptions` _can_ be used to enable rich text, but also for other things (essentially providing more options to Tippy.js.

We should be clear on this, and probably even link to the [All Props](https://atomiks.github.io/tippyjs/v6/all-props/) page of Tippy.

I would imagine something like this:
- explain what `@extraTippyOptions` can do/is useful for
- explain that we can't document all the possible options, so they should refer to the official API documentation for details
- explain that this can be used to enable rich text
- show a demo code snippet of using `allowHTML`
- explain that sanitization of the code is responsibility of the consumers